e2eshark/turbine-requirements.txt

```diff
@@ -0,0 +1,8 @@
+tabulate
+simplejson
+ml_dtypes
+sentencepiece
+accelerate
+auto-gptq
+optimum
+azure-storage-blob
```

Comment:
Some of these are already in https://github.com/nod-ai/SHARK-TestSuite/blob/main/e2eshark/requirements.txt. Want to split this into `ci-requirements.txt` and include just the extra packages? (Wouldn't want that file to pin to a specific version of one of these packages and that to possibly get out of sync with this file)

Reply:
Actually the point of this was to remove some of the requirements in the e2eshark requirements because it conflicts with the turbine requirements and causes pip problems. I will see if there is a way to use a requirements.txt pip install but ignore some of the requirements

Reply:
Oh... interesting. If you have logs or repro instructions someone else (me / others) could help debug that too.

Reply:
This won't be a problem in the future (nothing we should spend time debugging). It just has to do with different torch versions currently between e2eshark and turbine and the effects of that. Once torch 2.3 is in a good state for turbine, I will switch it over and we should be good :)